### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by cstillwellusgs**
#3

### CODE REVIEW

**Comment by cstillwellusgs** [comment]
Run the script titled "pipelines_targets_1.R" to execute all code. 

Do you prefer plotting in base R, ggplot2, or something different? Does it matter?

**Comment by cstillwellusgs** [comment]
Addresses #3

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@
 .Rhistory
 .RData
 .Ruserdata
+1_fetch/out
+2_process/out
+3_visualize/out
```
[the remark this answers is not in the document]
Comment (cstillwellusgs) [reply .gitignore]: Thanks for the suggestion!

File: 1_fetch/src/fetch_data.R [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Function to fetch data
+fetch_data <- function(sb_id, item_name, out_dir, out_file) {
+  destination <- file.path(out_dir, out_file)
+  item_file_download(sb_id, names = item_name, destinations = destination, 
+                     overwrite_file = TRUE) 
+}
```
Comment (jds485) [inline 1_fetch/src/fetch_data.R:6]: You should add a `return()` as the last line within all functions, or a comment that says what is returned. For `targets`, it helps to return an output file path or an R object (that will be explained in upcoming training issues). File path works well here for the download step

**Pull request merged**